MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include README.md
+include CHANGELOG.md
+include requirements.txt
```

Comment:
this is not needed.

Reply:
Is only requirements.txt not needed or everything?

Reply:
just requirements